pkg/kernel/Dockerfile

```diff
@@ -64,27 +65,11 @@ RUN case $(uname -m) in \
         fi && \
         cat /kernel_config${EXTRA} >> ${KERNEL_DEF_CONF}; \
     fi && \
-    make defconfig && \
-    make oldconfig && \
-    if [ -z "${EXTRA}" ]; then diff -cw .config ${KERNEL_DEF_CONF}; fi
+    cp ${KERNEL_DEF_CONF} .config && \
+    cp ${KERNEL_DEF_CONF} .config.new && \
+    make prepare
 
-# Kernel
-RUN make -j "$(getconf _NPROCESSORS_ONLN)" KCFLAGS="-fno-pie" && \
-    case $(uname -m) in \
-    x86_64) \
-        cp arch/x86_64/boot/bzImage /out/kernel; \
-        ;; \
-    aarch64) \
-        cp arch/arm64/boot/Image.gz /out/kernel; \
-        ;; \
-    esac && \
-    cp System.map /out && \
-    ([ "${EXTRA}" = "-dbg" ] && cp vmlinux /out || true)
-
-# Modules
-RUN make INSTALL_MOD_PATH=/tmp/kernel-modules modules_install
-
-# Out-of-tree, open source modules
```

Comment:
This code is still out of tree, so don't understand why you removed that comment.

Reply:
I changed it on "Prepare built-in ZFS" :) 
```
-# Out-of-tree, open source modules
+# Prepare built-in ZFS
```
because now ZFS is a bit special and runs its own procedure (with ./copy-builtin; etc).